src/components/post/DeletePostDialog.tsx

```diff
@@ -23,20 +18,22 @@ export default function DeletePostDialog(props: props) {
         onClose={props.handleCloseModal}
       >
         <DialogTitle>
-          <Typography variant="h4">Are you sure?</Typography>
+          <Box display="flex">
+            <Typography variant="body1">Are you sure to{"\u00A0"}</Typography>
+            <Typography variant="body1" color="error">
+              delete
+            </Typography>
+            <Typography variant="body1">{"\u00A0"}this post ?</Typography>
+          </Box>
         </DialogTitle>
         <DialogContent dividers>
-          <Typography>
-            Do you really want to delete the post? This process cannot be undone.
+          <Typography variant="body2" color="error">
+            *This action cannot be undone.
           </Typography>
         </DialogContent>
         <DialogActions>
-          <Button variant="contained" color="inherit" onClick={props.handleCloseModal}>
-            cancel
-          </Button>
-          <Button variant="contained" color="error" onClick={props.deletePost}>
-            delete
-          </Button>
+          <NormalButton label="Delete" color={COLOR.ERROR} onClick={props.deletePost} />
+          <NormalButton label="Cancel" onClick={props.handleCloseModal} />
```

Comment:
remove `Cancel` button and change it to `X` sign on top right ka

Reply:
Does this design inherit all popup dialog? If so, I'll create the public component for this.

Reply:
yes except for `Rate` and `Add advertisement` which probably not in this sprint

Reply:
Solved